### PR TITLE
[pxr] fix for export of namespaced point instancers

### PR DESCRIPTION
**Pull request by elrond79**
Also added tests for instancer / namespaced instancers

### CODE REVIEW

**Comment by kxl-adsk** [comment]
@elrond79 This is another change we prefer to keep post #154

**Comment by kxl-adsk** [comment]
@elrond79 Please resolve conflicts.

**Comment by elrond79** [comment]
Done...

File: lib/usd/translators/instancerWriter.cpp [hunk 1]
```diff
@@ -293,8 +293,14 @@ PxrUsdTranslators_InstancerWriter::writeInstancerAttrs(
 
             // Prototype names are guaranteed unique by virtue of having a
             // unique numerical suffix _# indicating the prototype index.
+
+            std::string sourceName(sourceNode.name().asChar());
+            if (_writeJobCtx.GetArgs().stripNamespaces){
```
[the remark this answers is not in the document]
Comment (elrond79) [reply lib/usd/translators/instancerWriter.cpp]: Think this should fix that error.  Odd that GetArgs and GetUsdStage not being publicly exposed hadn't caused problems before!

**Pull request merged**